pr2eus/robot-interface.l

```diff
@@ -454,6 +455,7 @@
          (dolist (av avs)
            (send robot :angle-vector av)
            (setq end-coords-current (mapcar #'(lambda (limb) (send robot limb :end-coords :copy-worldcoords)) limbs))
+	   (setq diff (v- (v- av (setq av (v+ av-prev (send self :sub-angle-vector av av-prev)))) diff-prev))
```

Comment:
@Affonso-Gui I have added your code to https://github.com/jsk-ros-pkg/jsk_pr2eus/pull/326, but not sure about the logic of this code.

I thought...
`setq av..` update `av` from `avs` to `av-prev + sub-angle-vector`, so if there is no 180 rotation this new `av` is `av`. Let set this new av as `new-av`.
Then, we calculate `av - new-av - diff-prev`, If `new-av` is equal to `av`, then `diff` is `- diff-prev`, Where did I go wrong?

Reply:
@k-okada It's exactly as you said, when there is no 180 turns `diff` is set to `- diff-prev`, meaning to undo last rotation.

For example, in an `av` sequence from 
```
0 → 1000 → 0
```
the `new-av` sequence becomes:
```
0 → - 80 → 0
```
Which is then used to set `midpoint` for IK.
In this case the `diff` becomes:
```
  +1080 , -1080
```
Which is then slowly added to `prev-av` and to the IK result (Line 484) in order to generate something like:
```
0 → 200 → 500 → 800 → 1000 → 800 → 500 → 200 → 0 
```